### ISSUE TITLE
Excercise the following approaches and techniques in the notebook

### ISSUE THREAD

**Issue by marc-thieme** [llamadeus/ai-industry#1]
[ ] KDE (01-ad-de/3)
[ ] Devise a cost model (01-ad-de/4)
[ ] Autocorrelation (Pearson Correlation Coefficient) (01-ad-de/5)
[ ] Pick window length where autocorrelation is low (<= 1) (01-ad-de/6)
[ ] Choose reasonable bandwidth through grid search (01-ad-de/6)
[ ] Time dependent estimator (01-ad-de/7)
[ ] Multivariate distribution plot (01-ad-de/7)
[ ] Ensemble model (01-ad-de/8)
[ ] Build estimators for one time value (01-ad-de/8)
    - optimize bandwidth
    - generate the signal
    - threshold optimization and evaluation
[ ] Gaussian mixture models (02-ad-hd/2)
[ ] Autoencoder (02-ad-hd/4)
[ ] RMSE for measuring accuracy of approach for filling missing values (03-sf-gp/2)
    - forward/backward
    - linear
    - nearest
    - polynomial
    - spline
[ ] Gaussian processes (03-sf-gp/3)
[ ] Standardization (for each column) (04-rul/1)
[ ] Heatmap (04-rul/1)
[ ] Linear regression model (04-rul/2)
[ ] Convolutional neural network (04-rul/2)
[ ] RUL prediction as a classifier (04-rul/3)
[ ] Surrogate-based optimization (e.g., hyper-parameter training) (04-rul/4)
[ ] Try binning (05-pm/1)
[ ] Class rebalancing methods (05-pm/2)
